gulpfile.ts

```diff
@@ -43,8 +45,10 @@ gulp.task('build.e2e', done =>
 gulp.task('build.prod', done =>
   runSequence('clean.prod',
               'tslint',
+              'css-lint',
```

Comment:
Would you name the file with `_` instead if dash or better remove the dash at all, in order to follow the convention in the rest of the tasks.